src/camera/camera3d.jl

```diff
@@ -297,10 +297,15 @@ function add_translation!(scene, cam::Camera3D)
     end
 
     on(camera(scene), scene.events.scroll) do scroll
-        if is_mouseinside(scene) && ispressed(scene, scroll_mod[])
+        if is_mouseinside(scene)
+          if ispressed(scene, scroll_mod[])
             zoom_step = (1f0 + 0.1f0 * zoomspeed[]) ^ -scroll[2]
             zoom!(scene, cam, zoom_step, shift_lookat[], cad[])
-            return Consume(true)
+          else
+            camspeed = compute_diff([scroll[1], scroll[2]])
+            translate_cam!(scene, cam, 10f0 * Vec3f(-camspeed[1], camspeed[2], 0f0))
+          end
+          return Consume(true)
```

Comment:
I think we should adjust this to be zoom without a scroll mod and translation with one. I think that makes more sense and matches up better with Axis.

Reply:
Not sure I understand -- the way this is implemented if the scroll mod default of `true` is untouched then the `ispressed` call returns `true` and you get the existing behavior of zooming on scroll. Changing the scroll mod to something else for example as I described earlier is what enables "panning" (translating) with scrolling.

As far as matching with Axis goes, I'm not convinced that it always makes sense to have the behaviors match. Even in testing it out I'm not sure I'd always want to use scroll to translate in a 3D view even though I probably almost always would in a 2D view since the latter feels very natural. Still, it's nice to have the option either way.

Reply:
I see -- so to put it in other terms this would basically be adding the ability to translate on scroll ("special behavior") by holding down a key, e.g. shift, in addition to the existing behavior of holding the right mouse button down to translate. Either way just using scroll without a modifier would work the same as it does today.

I have no issue with that, and honestly adding anything to the 3D side of things was a bit of an afterthought. 2D panning with scroll is really the main idea. So maybe it would be better to just remove the 3D part of the PR.

Reply:
Update - I went ahead and removed the 3D camera changes. @ffreyer (and others) any further comments?